packages/ssr-web/app/routes/index.ts

```diff
@@ -0,0 +1,82 @@
+import Route from '@ember/routing/route';
+import '../css/pay.css';
+import { inject as service } from '@ember/service';
+import * as Sentry from '@sentry/browser';
+import AppContextService from '@cardstack/ssr-web/services/app-context';
+import config from '../config/environment';
+import { generateMerchantPaymentUrl, gqlQuery } from '@cardstack/cardpay-sdk';
+
+interface UserSpaceRouteModel {
+  id: string;
+  name: string;
+  backgroundColor: string;
+  textColor: string;
+  paymentURL: string;
+}
+
+export default class UserSpaceRoute extends Route {
+  @service('app-context') declare appContext: AppContextService;
+
+  async model(): Promise<UserSpaceRouteModel> {
+    if (this.appContext.currentApp === 'card-space') {
+      try {
+        const cardSpaceResult: {
+          included: any[];
+        } = await (
+          await fetch(
+            `${config.hubURL ?? 'http://localhost:3000'}/api/card-spaces/${
```

Comment:
hubURL in local dev is not set. We can't use `environment === 'development'`, so we need to find something else to conditionally set hubURL to `localhost:3000`.

Reply:
`environment.js` should be modified so that the absence of `SSR_WEB_ENVIRONMENT` or `SSR_WEB_ENVIRONMENT` not being staging or production will replace `environment === 'development'`.